### DISCUSSION TITLE
Has anyone used stable diffusion for face obfuscation? So you look like yourself - but face recognition models don't recognize you. So you can safely use those images on social networks without the fear of being doxed. What are the best tools for that? Is Foocus UI a good choice for it?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [comfyanonymous/ComfyUI#2290] [General]
Has anyone used stable diffusion for face obfuscation? So you look like yourself - but face recognition models don't recognize you. So you can safely use those images on social networks without the fear of being doxed. What are the best tools for that? Is foocus UI a good choice?

Can you share your experience with it - what Loras did you use, prompts, and so on?

**Comment by ltdrdata** [comment]
If you run FaceDetailer with denoise around 0.5, your face will likely transform into something other than your actual face.
Maintaining your original face becomes even more challenging in the i2i process.

**Reply by martinenkoEduard** [reply]
But how to make it generate similar but not quite the same face?

The goal is to modify your own face so you look like yourself, and can share this image on social networks, but you can't be found (doxed) by using face search algorithms.

in practical terms - I will test it on my local face recognition software - something like ArcFace, AdaFace.
Maybe using compreface software.

**Reply by Woisek** [reply]
Look here -> https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/14303#discussioncomment-7862611

**Reply by ltdrdata** [reply]
Try using the Mediapipe detector to modify only certain parts of the face once.
https://github.com/ltdrdata/ComfyUI-extension-tutorials/blob/Main/ComfyUI-Impact-Pack/workflow/mediapipe-detector.png

**Reply by martinenkoEduard** [reply]
is there a workflow file?

**Reply by ltdrdata** [reply]
That png contains workflow. You can drag&drop to ComfyUI.
